src/compositor/hwcimage.cpp

```diff
@@ -66,7 +66,7 @@ class HwcImageLoadRequest : public QRunnable, public QEvent
             // Apply glass..
             if (effect.contains(QStringLiteral("glass"))) {
                 QImage glass("//usr/share/themes/jolla-ambient/meegotouch/icons/graphic-shader-texture.png");
-                glass = glass.scaled(glass.width() * pixelRatio, glass.height() * pixelRatio,
```

Comment:
You need to increase its size relative to the pixelRation. Otherwise, you it will be tiny and look different from that in the covers and the original app background.